news/10950.bugfix.rst

```diff
@@ -0,0 +1 @@
+Disable brotli import in vendored urllib3.
```

Comment:
Could we record _why_ brotli import is disabled in the news? It would save potential controbutors some clicks to figure out the reasoning if this ever become an issue.

Reply:
to be able to uninstall brotli itself?

Reply:
Yeah something like that. Let me just add it I guess…